### PR TITLE
feat:adding variables css for page

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyen-agilityio @hungnguyenhuu Please help me to check PR and merge it.

### CODE REVIEW

File: practice-two/src/assets/css/utils/variables.css [hunk 1]
```diff
@@ -16,4 +16,8 @@
   --dark-color: #111111;
   --light-color: #ffffff;
   --shadow-color: #3145f419;
+  --shadow-sm-color: #11111a1a;
+
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/utils/variables.css:20]: Give comment for the variables section you defined below.

Comment (LeVHoangduc) [reply practice-two/src/assets/css/utils/variables.css:20]: Because it has opacity and i named for it 'sm', that means a lighter

Comment (LeVHoangduc) [reply practice-two/src/assets/css/utils/variables.css:20]: @hungnguyenhuu

File: practice-two/src/assets/css/utils/variables.css [hunk 2]
```diff
@@ -16,4 +16,12 @@
...
+  --width-81: 81%;
+  --width-90: 90%;
+  --width-full: 100%;
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/utils/variables.css:23]: The naming is not good. Pls update

File: practice-two/src/assets/css/utils/variables.css (hunk 1 above)
Comment (hungnguyenhuu) [reply practice-two/src/assets/css/utils/variables.css:20]: I mean you should give the comment to describe the block of coding.

File: practice-two/src/assets/css/utils/variables.css (hunk 2 above)
Comment (LeVHoangduc) [reply practice-two/src/assets/css/utils/variables.css]: yah sure, i have updated it.

File: practice-two/src/assets/css/utils/variables.css (hunk 1 above)
Comment (LeVHoangduc) [reply practice-two/src/assets/css/utils/variables.css:20]: Yah, i will do that

**Pull request merged**